event_log.c

```diff
@@ -110,10 +112,19 @@ typedef struct __packed {
     /* u8 event[]; */
 } tpm12_event_t;
 
+/* The same as TPML_DIGEST_VALUES but little endian, as event log expects it */
+typedef struct __packed {
+    u32 count;
+    u16 sha1_id;
+    u8 sha1_hash[20];
+    u16 sha256_id;
+    u8 sha256_hash[32];
```

Comment:
Might as well use constants here too:
```suggestion
    u8 sha1_hash[SHA1_DIGEST_SIZE];
    u16 sha256_id;
    u8 sha256_hash[SHA256_DIGEST_SIZE];
```

Reply:
Found few other places in this file where constants can be used, changed those as well.

https://github.com/TrenchBoot/secure-kernel-loader/compare/f2a18cb10d6548d177ca1eff9b8334344b2a306b..b9aaa5c37b91adeed72a067dde349129b9970fde#diff-7aae9b14e1b9cc986e1cf987c539eebb9634a4bb0d1d55d59ec0f65a15fe82cc